diff --git a/CMakeLists.txt b/CMakeLists.txt
index 30bdeb5..a657741 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -78,7 +78,7 @@ ExternalProject_Add(BearSource
             -DCMAKE_MODULE_LINKER_FLAGS:STRING=${CMAKE_MODULE_LINKER_FLAGS}
             -DROOT_INSTALL_PREFIX:PATH=${CMAKE_INSTALL_PREFIX}
             ${CMAKE_CACHE_ARGS_EXTRA}
-        TEST_BEFORE_INSTALL
+        TEST_EXCLUDE_FROM_MAIN
             1
         TEST_COMMAND
             ctest # or `ctest -T memcheck`
@@ -95,7 +95,7 @@ if (ENABLE_FUNC_TESTS)
                 -DCMAKE_INSTALL_LIBDIR:PATH=${CMAKE_INSTALL_LIBDIR}
                 -DCMAKE_INSTALL_BINDIR:PATH=${CMAKE_INSTALL_BINDIR}
                 -DSTAGED_INSTALL_PREFIX:PATH=${STAGED_INSTALL_PREFIX}
-            TEST_BEFORE_INSTALL
+            TEST_EXCLUDE_FROM_MAIN
                 1
             INSTALL_COMMAND
                 ""
